src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -82,7 +81,7 @@ public CheckResult Check(string expressionText, RecordType parameterType = null)
 
             var formula = new Formula(expressionText);
 
-            formula.EnsureParsed(TexlParser.Flags.None);
+            formula.EnsureParsed(GetTexlParserFlags());
```

Comment:
>GetTexlParserFlags [](http://example.com/codeflow?start=33&length=18)

fyi, This will get superseded by this PR for exposing parseresult: [https://github.com/microsoft/Power-Fx/pull/319](https://github.com/microsoft/Power-Fx/pull/319)

Reply:
Good catch